lib/rest-adapter.js

```diff
@@ -571,16 +577,16 @@ RestAdapter.prototype.allRoutes = function() {
 };
 
 RestAdapter.prototype.getClasses = function() {
-  return this.remotes.classes(this.options).map(function(c) {
-    return new RestClass(c);
+  return this.remotes.classes(this.options).map(c => {
+    return new RestClass(c, this.options);
```

Comment:
Most of our packages allow ES6 style (as supported by Node.js 4.x) now. See [engines in package.json](https://github.com/strongloop/strong-remoting/blob/96042273024b8b8cc17d2576d6967cd4b089209a/package.json#L18) and http://loopback.io/doc/en/contrib/style-guide.html.

In general, we are keeping existing code as it is, but newly added lines can/should leverage ES6  syntax.